### ISSUE TITLE
Missing Clerk Secret Key. Go to https://dashboard.clerk.com and get your key for your instance.

### ISSUE THREAD

**Issue by jamalgillis** [panteliselef/astro-with-clerk-auth#181]
After going through the set up here https://clerk.com/changelog/2024-03-28 I'm getting this error.  I believe it's some astro error, just not sure where to start really.

**Comment by panteliselef** [comment]
Hey @jamalgillis did you replace
```sh
PUBLIC_ASTRO_APP_CLERK_PUBLISHABLE_KEY=<your-publishable-key>
CLERK_SECRET_KEY=<your-secret-key>
```
with the keys that were provided in [dashboard.clerk.com](https://dashboard.clerk.com) ?

**Comment by jamalgillis** [comment]
Yes i did.  I'm thinking maybe it has to do with my local .env variables not being read for some reason.  I don't fully understand how environment variables work in Astro.
